<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>956</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>939</width>
      <height>471</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,0,0,0,0">
     <item row="8" column="2">
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Total vehicles created</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="checkBox_4">
       <property name="text">
        <string> Yes/No</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string> Color of the Vehicle</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3" colspan="4">
      <widget class="QComboBox" name="comboBox_2">
       <item>
        <property name="text">
         <string>Electric</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Gasoline</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Diesel</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Hybrid</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Kerosene</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string> Motor</string>
       </property>
      </widget>
     </item>
     <item row="8" column="3" colspan="4">
      <widget class="QLineEdit" name="lineEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string> Name of the Vehicle</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLabel" name="label_13">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Wagons</string>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QSpinBox" name="spinBox_2">
       <property name="minimum">
        <number>5</number>
       </property>
       <property name="maximum">
        <number>20</number>
       </property>
       <property name="value">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>Yes/No</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QCheckBox" name="checkBox_2">
       <property name="text">
        <string>Yes/No       </string>
       </property>
      </widget>
     </item>
     <item row="7" column="2" colspan="5">
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QCheckBox" name="checkBox_7">
       <property name="text">
        <string>Yes/No</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3" colspan="2">
      <widget class="QSlider" name="horizontalSlider">
       <property name="minimum">
        <number>80</number>
       </property>
       <property name="maximum">
        <number>450</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="label_9">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string> Type of Fuel</string>
       </property>
      </widget>
     </item>
     <item row="9" column="3">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;The License is:   &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>  Wings                   </string>
       </property>
      </widget>
     </item>
     <item row="6" column="5">
      <widget class="QRadioButton" name="radioButton_2">
       <property name="text">
        <string>Punture Kit</string>
       </property>
      </widget>
     </item>
     <item row="5" column="6">
      <widget class="QCheckBox" name="checkBox_3">
       <property name="text">
        <string>Yes/No                </string>
       </property>
      </widget>
     </item>
     <item row="1" column="2" colspan="5">
      <widget class="QComboBox" name="comboBox_3">
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>40</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_12">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>  Locomotiv</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt; Select a License&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="9" column="4">
      <widget class="QLineEdit" name="lineEdit_4">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="5" colspan="2">
      <widget class="QSpinBox" name="spinBox">
       <property name="minimum">
        <number>80</number>
       </property>
       <property name="maximum">
        <number>450</number>
       </property>
      </widget>
     </item>
     <item row="0" column="2" colspan="5">
      <widget class="QLineEdit" name="lineEdit_2"/>
     </item>
     <item row="6" column="6">
      <widget class="QRadioButton" name="radioButton">
       <property name="text">
        <string>Spare Wheel</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string> Fuel</string>
       </property>
      </widget>
     </item>
     <item row="5" column="5">
      <widget class="QLabel" name="label_14">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Reactors              </string>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Generate Random License Plate</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QCheckBox" name="checkBox_6">
       <property name="text">
        <string>Yes/No</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Create Vehicle</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string> Number Wheels</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QComboBox" name="comboBox_4"/>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="checkBox_5">
       <property name="text">
        <string> Yes/No                </string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QLabel" name="label_15">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Undercarriage</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string> Power</string>
       </property>
      </widget>
     </item>
     <item row="6" column="4">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="9" column="5">
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>See the features</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2" colspan="5">
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>White</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Black</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Red</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Blue</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Yellow</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Green</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>956</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>spinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>557</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>777</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_4</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>lineEdit_4</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>468</y>
    </hint>
    <hint type="destinationlabel">
     <x>602</x>
     <y>472</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
